srcpkgs/syncthing-gtk/template

```diff
@@ -2,14 +2,16 @@
 pkgname=syncthing-gtk
 reverts="0.14.36_1"
 version=0.9.4.4
-revision=3
-build_style=python2-module
-hostmakedepends="python-setuptools"
-depends="syncthing python-dateutil libnotify librsvg python-bcrypt
- python-cairo gtk+3 python-gobject"
-short_desc="GTK3 and python based GUI for Syncthing"
+revision=4
+_ver=$version+ds+git20201209+c46fbd8-1
+wrksrc="$pkgname-debian-$_ver"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="syncthing python3-dateutil libnotify librsvg python3-bcrypt
+ python3-cairo gtk+3 python3-gobject"
+short_desc="GTK based GUI for Syncthing"
 maintainer="Frank Steinborn <steinex@nognu.de>"
 license="GPL-2.0-or-later"
-homepage="https://github.com/syncthing/syncthing-gtk"
-distfiles="https://github.com/syncthing/syncthing-gtk/archive/v${version}.tar.gz"
-checksum=896ddaaba4ad0b8f090c5a381a28b3da759932314562cdd50ca288543b03ddcc
+homepage="https://salsa.debian.org/debian/syncthing-gtk/"
+distfiles="https://salsa.debian.org/debian/syncthing-gtk/-/archive/debian/$_ver/syncthing-gtk-debian-$_ver.tar.gz"
```

Comment:
```suggestion
distfiles="$DEBIAN_SITE/main/s/syncthing-gtk/syncthing-gtk_${version}.orig.tar.xz"
```
With version change to `0.9.4.4+ds+git20201209+c46fbd8`

Reply:
The version change is a bit ugly:
```
shellcheck    update    0.7.2_2           0.8.0_1                2712KB 
soundtouch    update    2.2_1             2.3.1_1                81KB 
syncthing-gtk update    0.9.4.4_4         0.9.4.4+ds+git20201209+c46fbd8_1 - 
vscode        update    1.62.0_1          1.62.1_1               19MB
```